CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing to Stream Chat Swift SDK
+
+We're glad you want to contribute to the Stream team 🎉
+
+---
+
+_So you.._
+
+## Got stuck on something 💭
+
+Please check [stackoverflow](https://stackoverflow.com/questions/tagged/getstream-io) and ask your questions there.
+If your question is not generic, you can send us a [support request](https://getstream.io/support).
+
+## Found a bug 🐞
+
+Please create a github issue with as much info as possible (follow the Issue Template closely).
+
+## Have a feature request 📈
+
+Please create a github issue with as much info as possible.
+
+## Fixed a bug 🩹
+
+Please open a PR with as much info as possible: clear description of the problem and the solution.
+Include the relevant github issue number if applicable. 
+
+Make sure Changelog is updated correspondingly (we'll probably change wording but it'll help us immensely)
+
+Before submitting, please make sure you're finished with the PR (and all tests pass) and do not make changes until it's reviewed.
+
+## Implemented or changed a feature 🌈
+
+Guidelines on "Fixed a bug" part is applicable.
+
+## Prepared a new Release 🚀
+
+Please follow [release guide](https://github.com/GetStream/stream-chat-swift/blob/master/releasing.md) closely.
+Make sure to follow all QA test procedure for minor and major releases. 
```

Comment:
The contributors can't make a release.

Reply:
I think it's good to describe how we are making a new release.

Reply:
Agreed, release can only be made by Stream employees. Where would you suggest I add this info? In another file? Or maybe I can change the text so it reads "If you're a Stream employee and want to make a release..." so it's clear this is intended for Stream employees.

On "How to make a release" it's explained in the `releasing.md` file, isn't it?

Reply:
I think it's obvious that Stream employees only can make a release. So no need to describe how to make a release for contributors.

Reply:
I've refactored it so it's clear contributors can't make a release, and this paragraph states how Stream employees make releases.